### PR TITLE
chore(debug): add tab and bot debug handler

### PR DESCRIPTION
**Pull request by kuojianlu**

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -506,6 +506,8 @@
   "error.apiConnector.componentNotExistError": "Component %s not exist, please add first",
   "error.apiConnector.envVarExistError": "Please provide a different API name to avoid conflicts with existing env variables %s in .env.teamsfx.local",
   "error.debugHandler.InvalidSSODebugArgsError": "objectId, clientId and clientSecret are all required if you use existing AAD app for SSO.",
+  "error.debugHandler.InvalidBotDebugArgsError": "botId and botPassword are all required if you use existing bot. botMessagingEndpoint is required.",
```
Comment (swatDong) [inline packages/fx-core/resource/package.nls.json:509]: Separate into two errors:
- "botMessagingEndpoint is required."
- "botId and botPassword need to be set together if you use existing bot."

File: packages/fx-core/src/component/debugHandler/bot.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  Result,
+  FxError,
+  ok,
+  err,
+  ProjectSettingsV3,
+  v3,
+  Void,
+  assembleError,
+} from "@microsoft/teamsfx-api";
+import { loadProjectSettingsByProjectPath } from "../../core/middleware/projectSettingsLoader";
```
Comment (swatDong) [inline packages/fx-core/src/component/debugHandler/bot.ts:19]: [Code Style] - ensure alphabetical order of import paths, for easier maintenance.

File: packages/fx-core/src/component/debugHandler/tab.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    } catch (error: any) {
+      return err(assembleError(error, errorSource));
+    }
+  }
+
+  private async checkArgs(): Promise<Result<boolean, FxError>> {
+    if (!this.args.baseUrl) {
+      return err(InvalidTabDebugArgsError());
+    }
+    const pattern = /https:\/\/localhost:\d+/;
```
Comment (swatDong) [inline packages/fx-core/src/component/debugHandler/tab.ts:85]: Will we support `http` (without ssl cert)?

Comment (kuojianlu) [reply packages/fx-core/src/component/debugHandler/tab.ts:85]: No. The content url in manifest have to be `https`.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (kuojianlu) [reply packages/fx-core/resource/package.nls.json]: Updated.

File: packages/fx-core/src/component/debugHandler/bot.ts (hunk 2 above)
Comment (kuojianlu) [reply packages/fx-core/src/component/debugHandler/bot.ts]: Nice idea. Updated.

**Review by swatDong — APPROVED** [review]

**Pull request merged**